schema/samcli.json

```diff
@@ -23,9 +23,149 @@
               "title": "Parameters for the init command",
-              "description": "Available parameters for the init command:\n",
+              "description": "Available parameters for the init command:\n* no_interactive:\nDisable interactive prompting for init parameters. (fail if any required values are missing)\n* architecture:\nArchitectures for Lambda functions.\u001b[1m\n\nArchitectures: ['arm64', 'x86_64']\u001b[0m\n* location:\nTemplate location (git, mercurial, http(s), zip, path).\n* runtime:\nLambda runtime for application.\u001b[1m\n\nRuntimes: dotnet6, go1.x, java17, java11, java8.al2, java8, nodejs18.x, nodejs16.x, nodejs14.x, nodejs12.x, provided, provided.al2, python3.9, python3.8, python3.7, python3.11, python3.10, ruby3.2, ruby2.7\u001b[0m\n* package_type:\nLambda deployment package type.\u001b[1m\n\nPackage Types: Zip, Image\u001b[0m\n* base_image:\nLambda base image for deploying IMAGE based package type.\u001b[1m\n\nBase images: amazon/go1.x-base, amazon/go-provided.al2-base, amazon/java8.al2-base, amazon/nodejs18.x-base, amazon/nodejs12.x-base, amazon/python3.7-base, amazon/ruby3.2-base, amazon/python3.8-base, amazon/python3.10-base, amazon/java17-base, amazon/dotnet6-base, amazon/java8-base, amazon/python3.11-base, amazon/nodejs14.x-base, amazon/nodejs16.x-base, amazon/ruby2.7-base, amazon/python3.9-base, amazon/java11-base\u001b[0m\n* dependency_manager:\nDependency manager for Lambda runtime.\u001b[1m\n\nDependency managers: bundler, maven, mod, gradle, npm, pip, cli-package\u001b[0m\n* output_dir:\nDirectory to initialize AWS SAM application.\n* name:\nName of AWS SAM Application.\n* app_template:\nIdentifier of the managed application template to be used. Alternatively, run '$sam init' without options for an interactive workflow.\n* no_input:\nDisable Cookiecutter prompting and accept default values defined in the cookiecutter config.\n* extra_context:\nOverride custom parameters in the template's cookiecutter.json configuration e.g. {\"customParam1\": \"customValue1\", \"customParam2\":\"customValue2\"}\n* tracing:\nEnable AWS X-Ray tracing for application.\n* application_insights:\nEnable CloudWatch Application Insights monitoring for application.\n* beta_features:\nEnable/Disable beta features.\n* debug:\nTurn on debug logging to print debug message generated by AWS SAM CLI and display timestamps.",
               "type": "object",
-              "properties": {}
+              "properties": {
+                "no_interactive": {
+                  "title": "no_interactive",
+                  "type": "boolean",
+                  "description": "Disable interactive prompting for init parameters. (fail if any required values are missing)"
+                },
+                "architecture": {
+                  "title": "architecture",
+                  "type": "string",
+                  "description": "Architectures for Lambda functions.\u001b[1m\n\nArchitectures: ['arm64', 'x86_64']\u001b[0m",
```

Comment:
Will the unicode characters show up correctly in tool tips?

Reply:
It shows up like this in VSCode: 
![image](https://github.com/aws/aws-sam-cli/assets/51037424/ce1b30bc-3f97-4268-b026-4c02e3dbed16)
Based on the command line output, it seems like it's supposed to be bolded, but it's not appearing that way. For readability sake, and because different IDEs could process markdown differently, we could opt to remove instances of the `[1m` and `[0m` altogether?

Reply:
Lets strip out the ANSI encoding from the text, not sure if there is a built in way of doing that so that we can avoid having manually replace things.